### PR TITLE
OCPBUGS-19994:updates verification step for CLI NMState Operator

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OCPBUGS-19994

Link to docs preview:
[Preview](https://72655--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator#installing-the-kubernetes-nmstate-operator-CLI_k8s-nmstate-operator)

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @qiowang721

### CODE REVIEW

**Comment by qiowang721** [comment]
Hi @JoeAldinger  We need to ensure the clusterserviceversion is in Succeeded first, then create instance of nmstate operator.
So should keep the original verification step for `clusterserviceversion`, and move the original step4 behind this verification.

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 1]
```diff
@@ -90,13 +93,33 @@ EOF
 +
 [source,terminal]
 ----
-oc get clusterserviceversion -n openshift-nmstate \
- -o custom-columns=Name:.metadata.name,Phase:.status.phase
+oc get pod -n openshift-nmstate
```
Comment (JoeAldinger) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:96]: @qiowang721 does this work?

Comment (qiowang721) [reply modules/k8s-nmstate-deploying-nmstate-CLI.adoc:96]: @JoeAldinger sorry maybe i did not describe clearly...
We need to keep the original Verification section as it is, the Succeeded csv means the operator is installed.
For the original step4, it should be a step after Verification, and we can use `oc get pod` or `oc get deployment` `oc get daemonset` to verify it.

**Comment by qiowang721** [comment]
@JoeAldinger  LGTM, thanks.

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 2]
```diff
@@ -72,7 +72,22 @@ spec:
 EOF
 ----
 
-. Create instance of the `nmstate` operator:
+. Confirm the `ClusterServiceVersion` (CSV)status for the `nmstate` Operator deployment `Succeeded`:
```
Comment (dfitzmau) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:75]:
```suggestion
. Confirm the `ClusterServiceVersion` (CSV) status for the `nmstate` Operator deployment equals `Succeeded`:
```

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 3]
```diff
@@ -72,7 +72,22 @@ spec:
...
++
+[source,terminal,subs="attributes+"]
+----
+oc get clusterserviceversion -n openshift-nmstate \
```
Comment (dfitzmau) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:79]:
```suggestion
$ oc get clusterserviceversion -n openshift-nmstate \
```

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 4]
```diff
@@ -72,7 +72,22 @@ spec:
...
+ -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
++
+.Example output
```
Comment (dfitzmau) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:83]: Tedious formatting will be needed to bring the "Phase" heading over the "Succeeded" state.

Screenshot from 2024-03-08 12-53-36

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 5]
```diff
@@ -84,19 +99,26 @@ metadata:
 EOF
 ----
 
-.Verification
-
-* Confirm that the deployment for the `nmstate` operator is running:
+. Verify the pods for NMState Operator are running:
```
Comment (dfitzmau) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:102]: Here the Operator is prepended with "NMState" while elsewhere it's `nmstate`. Best to stick with one format for consistency.

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 6]
```diff
@@ -84,19 +99,26 @@ metadata:
...
- -o custom-columns=Name:.metadata.name,Phase:.status.phase
+oc get pod -n openshift-nmstate
 ----
 +
 .Example output
 [source, terminal,subs="attributes+"]
 ----
-Name                                             Phase
-kubernetes-nmstate-operator.{product-version}.0-202210210157   Succeeded
+NAME                     READY  STATUS  RESTARTS  AGE
```
Comment (dfitzmau) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:112]: Heading and formatting :banana: :
Screenshot from 2024-03-08 12-56-32

**Comment by dfitzmau** [comment]
Hi @JoeAldinger . Nice work. I added comments inline.

/remove-label peer-review-in-progress
/label peer-review-done

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc (hunk 5 above)
Comment (JoeAldinger) [reply modules/k8s-nmstate-deploying-nmstate-CLI.adoc:102]: Here I'm referring to the Operator in a general sense like it is done in the introductory sentence; whereas, in other spots `nmstate` is referring to the Operators namespace or other fields. In short, I was trying to match the consistency of the current docs, which switch between the two.

Comment (dfitzmau) [reply modules/k8s-nmstate-deploying-nmstate-CLI.adoc:102]: Ah, thanks for clarifying.

**Pull request merged**